subsys/fs/littlefs_fs.c

```diff
@@ -20,7 +20,9 @@
 #include <fs/littlefs.h>
 #include <drivers/flash.h>
 #include <storage/flash_map.h>
-
+#if IS_ENABLED(CONFIG_FS_LITTLEFS_BLK_DEV)
+#include <storage/disk_access.h>
```

Comment:
Why is include conditional?

Reply:
It is conditional to not pollute the `littlefs_fs.c` file when we don't use littlefs on the block device (like SD card).

Reply:
What, how does it pollute? Conditional include is not necessary here.

Reply:
I wanted to avoid adding disk related API stuff (even the header) when only `littlefs` with flash support is used.
I do agree, that it will not harm if we add this header unconditional.

Reply:
Remove the ifdef around include.